tests/SchemaTest.php

```diff
@@ -149,35 +150,37 @@ public function testDropIndex(): void
         });
 
         $index = $this->getIndex('newcollection', 'field_a_1_field_b_1');
-        $this->assertFalse($index);
+        $this->assertNull($index);
 
+        $indexName = 'field_a_-1_field_b_1';
         Schema::table('newcollection', function ($collection) {
             $collection->index(['field_a' => -1, 'field_b' => 1]);
         });
 
-        $index = $this->getIndex('newcollection', 'field_a_-1_field_b_1');
+        $index = $this->getIndex('newcollection', $indexName);
         $this->assertNotNull($index);
```

Comment:
Here the assertion was always true, `$index` was `false` for missing index.

Reply:
Having `assertIndexExists()` and `assertIndexNotExists()` would make the tests more readable if you wanted to go that route. I realize you can't get rid of `getIndex()` entirely, since some later tests expect to make assertions on the IndexInfo structure.

Reply:
Thanks for the suggestion, that's more explicit with the new assert methods.